### ISSUE TITLE
Handling mid-page loads

### ISSUE THREAD

**Issue by BenRacicot** [jlmakes/scrollreveal#552]
I've noticed when my page loads at a certain scroll location, like via fragment or when the user uses back button to come back to me page, that section does not trigger a reveal. Also all folds above it are not revealed.

This causes confusion when the user must scroll slightly to trigger reveals up or down.
Is there a configuration option or recommended way to alleviate this UX?

**Comment by xyeres** [comment]
I'm debugging a site for a client that used ScrollReveal and this is the issue they've had. I haven't done a lot of digging into the docs but I suspect this is an active issue due to this still being open?

**Comment by BenRacicot** [comment]
Yeah I don't think this project is active at all. There's another major issue still open about scroll direction reveals too which is a show-stopper.

**Comment by xyeres** [comment]
Thanks for the input!
